packages/inventory/src/Stocktake/ListView/ListView.tsx

```diff
@@ -0,0 +1,114 @@
+import React, { FC, useState } from 'react';
+import { useNavigate } from 'react-router';
+import {
+  DataTable,
+  useColumns,
+  useListData,
+  TableProvider,
+  createTableStore,
+  useNotification,
+  generateUUID,
+  useOmSupplyApi,
+} from '@openmsupply-client/common';
+import { NameSearchModal } from '@openmsupply-client/system/src/Name';
+import { Toolbar } from './Toolbar';
+import { AppBarButtons } from './AppBarButtons';
+import { getStocktakeListViewApi } from './api';
+import { StocktakeRow } from '../../types';
+
+export const StocktakeListView: FC = () => {
+  const navigate = useNavigate();
+  const { error } = useNotification();
+  const { api } = useOmSupplyApi();
+
+  const {
+    totalCount,
+    data,
+    isLoading,
+    onDelete,
+    // onUpdate,
+    sortBy,
+    onChangeSortBy,
+    onCreate,
+    onChangePage,
+    pagination,
+    filter,
+    invalidate,
+  } = useListData(
+    {
+      initialSortBy: { key: 'comment' },
+    },
+    'invoice',
+    getStocktakeListViewApi(api)
+  );
+
+  const columns = useColumns<StocktakeRow>(
+    [
+      'stocktakeDate',
+      'stocktakeNumber',
+      'comment',
+      'description',
+      'status',
+      'selection',
+    ],
+    { onChangeSortBy, sortBy },
+    [sortBy]
+  );
+
+  const [open, setOpen] = useState(false);
+
+  return (
+    <>
+      <NameSearchModal
```

Comment:
I'll write up the stocktake requirements tomorrow and have a think about what should happen here

Reply:
Ah oops. I've removed the modal for now. I suspect this will just be a way to create some filtered list of items.. which with the current filtering won't be ideal 🤔